include/dblayer/mysql.php

```diff
@@ -182,7 +182,7 @@ function close()
 	{
 		if ($this->link_id)
 		{
-			if ($this->query_result)
+			if (is_object($this->query_result))
```

Comment:
Per the PHP manual, it returns an object, but if you think `is_resource` is better then this can be changed.
```Returns FALSE on failure. For successful SELECT, SHOW, DESCRIBE or EXPLAIN queries mysqli_query() will return a mysqli_result object. For other successful queries mysqli_query() will return TRUE.```